lib/src/components/NxVulnerabilityIndicator/NxVulnerabilityIndicator.scss

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @use '../../scss-shared/nx-dark-mode-helpers';
+ 
+// eyeballed, assumes $nx-font-size surrounding text
+$-vertical-alignment: -0.1em;
```

Comment:
I'll want to come back to this once I see the mockups.  I suspect it is not correct